src/index.html

```diff
@@ -5,9 +5,244 @@
+                <li class="nav__link"><a class="nav__item" href="#home">Home</a></li>
+                <li class="nav__link"><a class="nav__item" href="#about-us">About us</a></li>
+                <li class="nav__link"><a class="nav__item" href="#gallery">Gallery</a></li>
+                <li class="nav__link"><a class="nav__item" href="#">Consultation</a></li>
+                <li class="nav__link"><a class="nav__item" href="#testimonials">Testimonials</a></li>
+                <li class="nav__link"><a class="nav__item" href="#contact-us">Contacts</a></li>
+              </ul>
+              <div class="nav__call">
+                <span class="nav__call-number">+1 234 555-55-5</span>
+                <span class="nav__call-text">Call to order</span>
+              </div>
+            </nav>
+          </div>
+          <div class="header__container">
+            <div class="header__top">
+              <div class="header__burger">
+                <span class="header__burger-icon"></span>
+              </div>
+              <div class="header__logo"></div>
+              <div class="header__phone">
+                <span class="header__phone-number">+1 234 555-55-5</span>
+              </div>
+            </div>
+            <h1 class="header__title">Miami<br>Condo Kings</h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="main__section">
+        <div class="about-us" id="about-us">
+          <h2 class="about-us__title">
+            Miami best
+            <br>
+            Real estate
+            <span class="about-us__title-filter"> here </span>
+          </h2>
+          <article class="about-us__article">
+            <h3 class="about-us__article-subtitle">Over 50 years experience in love.</h3>
+            <p class="about-us__article-text">
+              Mazel and VA Tsukkerman in his "Analysis of musical works."
+              Pointillism, which originated in the music of the early twentieth
+              century, microforms, found a distant historical.
+            </p>
+            <p class="about-us__article-text">
+              The market leader in real estate since 2003. Awards Real Estate
+              Company, 2010, 2011, 2013. More than 200 satisfied customers
+              premium segment. We focus only on new buildings and high-level
+              doskanalno know this area better than other specialists.
+              Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+            <p class="about-us__article-text-bold">
+              We will never be late!
+              <br>
+              Guaranteed secure transaction - 100%
+            </p>
+          </article>
+        </div>
+      </section>
+
+      <section class="main__section">
+        <div class="gallery" id="gallery">
+          <h2 class="gallery__title">Gallery</h2>
+          <div class="gallery__item">
```

Comment:
```suggestion
          <div class="gallery__item">
```
Why do you need this wrapper? You may just use 'gallery__photos'